src/functional_test_runner/lib/config/schema.js

```diff
@@ -88,6 +88,10 @@ export const schema = Joi.object().keys({
     elasticsearch: urlPartsSchema(),
   }).default(),
 
+  elasticsearchServerArgs: Joi.array(),
```

Comment:
Changing this to an object would allow us to directly pass options to `kbn/es` `installSnapshot`/`installSource`.

Thinking something shaped like this: 
```
{
    license:  'basic',
    password: 'changeme',
    esArgs: []
}
```

Reply:
Great, that's a good plan. But then they have a different appearance than the kibanaServerArgs. I guess that doesn't look too bad.

Reply:
Maybe the name should be something else, since `esArgs: []` within is now the array.